### DISCUSSION TITLE
Time Control Quick Create Editor not working on Yetiforce 6.2

### DISCUSSION THREAD

**Discussion by staples1347** [YetiForceCompany/YetiForceCRM#15278] [General]
On the Yetiforce 6.2 demo, Time Control is using a new Quick Create Editor that looks a lot better than the legacy editor. On a self hosted installation that I upgraded from Yetiforce 5.2 to Yetiforce 6.2, I am still getting the old Time Control interface. Also, in Standard Modules, when I click on Settings for Time Control and then click "Quick Create Editor", it throws an error: "Call to a member function getFieldInfo() on bool" . Is anyone else experiencing this problem?

This problem is occurring on our installation of Yetiforce 6.2

**Comment by rskrzypczak** [comment]
There is a parameter in the configuration file responsible for the layout in quick create, take a look at this:
https://github.com/YetiForceCompany/YetiForceCRM/blob/3f488e991501ba63ed0fce2db9401d9125f94541/config/Performance.php#L152-L153

As for the error, I can't help you. You need to enable debugging and provide more information. You should also check the update logs, because maybe the update was not fully performed correctly.

P.S. For future reference, do not create duplicate discussions (https://github.com/YetiForceCompany/YetiForceCRM/discussions/15272).
